.jenkins/actions/fetch_caches.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+BACKEND=$1
+EXPNAME=$2
+SANITIZED_BACKEND=`echo $BACKEND | sed 's/:/_/g'` #sanitize the backend from any ':'
+CACHE_DIR="/scratch/snx3000/olifu/jenkins/scratch/store_gt_caches"
+if [ ! -d $(pwd)/fv3gfs-physics/.gt_cache ]; then
+    if [ -d ${CACHE_DIR}/${EXPNAME}/${SANITIZED_BACKEND} ]; then
+        version_file=${CACHE_DIR}/${EXPNAME}/${SANITIZED_BACKEND}/GT4PY_VERSION.txt
+        if [ -f ${version_file} ]; then
+            version=`cat ${version_file}`
+        else
+            version=""
+        fi
+        if [ "$version" == "$GT4PY_VERSION" ]; then
+            if [ -d ${CACHE_DIR}/${EXPNAME}/${SANITIZED_BACKEND}/.gt_cache  ]; then
+                cp -r ${CACHE_DIR}/${EXPNAME}/${SANITIZED_BACKEND}/.gt_cache fv3gfs-physics/.
+                find fv3gfs-physics/. -name m_\*.py -exec sed -i "s|\/scratch\/snx3000\/olifu\/jenkins_submit\/workspace\/pace-physics-cache-setup\/backend\/${SANITIZED_BACKEND}\/experiment\/${EXPNAME}\/slave\/daint_submit/fv3gfs-physics|$(pwd)/fv3gfs-physics|g" {} +
```

Comment:
If you run the existing fetch_caches.sh script inside the fv3gfs-physics directory, it should behave the same as this edited copy. You may have some kind of error with the find command above, and that error might or might not cause a test failure, so be sure to fix that line as well, somehow. I don't really know what it does.

Reply:
From our discussion, it probably makes sense to do this find and replace in `generate_caches` to replace the path with a constant placeholder string (whatever you like), so that in this script we can replace the placeholder with the current directory.

Very strange that gt4py caches need to have a string referring to the directory the file is in. This should be possible to get programmatically.